packages/react-composites/src/composites/CallComposite/pages/CallPage.tsx

```diff
@@ -89,9 +95,28 @@ export const CallPage = (props: CallPageProps): JSX.Element => {
   const errorBarProps = usePropsFor(ErrorBar);
   const mutedNotificationProps = useSelector(mutedNotificationSelector);
   const networkReconnectTileProps = useSelector(networkReconnectTileSelector);
+  /* @conditional-compile-remove(dtmf-dialer) */
+  const remoteParticipantsConnected = useSelector(getRemoteParticipantsConnectedSelector);
+
+  /* @conditional-compile-remove(dtmf-dialer) */
+  const adapter = useAdapter();
+  /* @conditional-compile-remove(dtmf-dialer) */
+  const callees = adapter.getState().targetCallees;
+  /* @conditional-compile-remove(dtmf-dialer) */
+  const renderDtmfDialerFromStart = showDtmfDialer(callees);
+
+  /* @conditional-compile-remove(dtmf-dialer) */
+  const [dtmfDialerPresent, setDtmfDialerPresent] = useState<boolean>(renderDtmfDialerFromStart);
+  /* @conditional-compile-remove(dtmf-dialer) */
+  const [initialDialerDismissed, setInitialDialerDismissed] = useState<boolean>(false);
```

Comment:
consider a useRef for this property instead. We don't use this property to trigger re-renders. It is only used in the `if` to make sure we only auto-dismiss if the dialer was auto-shown.

(untested example):
```
// We show the dialer if we are in a call with a single callee.
// Track auto dismissing the dialer when a second participant joins the call.
const dialerShouldAutoDismiss = useRef<boolean>(renderDtmfDialerFromStart);

useEffect(() => {
  if(dialerShouldAutoDismiss.current && remoteParticipantsConnected > 1) {
    dialerShouldAutoDismiss.current = false;
    setDtmfDialerPresent(false);
  }
}, [remoteParticipantsConnected]);
```